redirectpage-tests/README.adoc

```diff
@@ -0,0 +1,27 @@
+
+The redirectPage.js implements the redirect flow of pages from sandbox developer catalog.
+
+== Test
+
+1. In the assets static folder install node_modules by running 
+    ```
+    npm i
+    ```
+2. To run tests, execute
+    ```
+    npm run test
+    ``` 
+
+== Implementation guidelines
+
+1. When user clicks on a catalog item on developers.redhat.com it redirects to this url format
+    `${baseUrl}?link="catalogName"&keyword="keyword"&selectedId="selectedId"`
+2. If the user is not authenticated, it redirects to sso page for user to login
+3. Once the user is authenticated it redirects to the page user wants to access which is in the format
+    `${consoleUrl}/${link}/ns/${defaultUserNamespace}?keyword="keyword"&selectedId="selectedId"` with
```

Comment:
Hm...
What URL should I use if I want to get redirected to
```
{consoleURL}/catalog/ns/alexeykazakov-dev?category=databases&selectedId=8ef58d8f-edcc-4284-96bb-5006360b6e73
```
Where `alexeykazakov-dev` is my default namespace.
What should I use for `keyword` to get `category=databases` as the result?

Reply:
In the google doc there is no URL with category? Am I missing something?

Reply:
Yeah, you are rigth.
But could you please clarify what URL should be used to access
```
{consoleURL}/catalog/ns/alexeykazakov-dev?keyword=jboss&selectedId=openshift-helm-charts--https%3A%2F%2Fgithub.com%2Fopenshift-helm-charts%2Fcharts%2Freleases%2Fdownload%2Fredhat-eap8-1.1.2%2Feap8-1.1.2.tgz
```
?
Should we use 
```

Reply:
Yes!

Reply:
Yes, that is correct